Zina_Yeghiazaryan/python/25_05_2021/Pascal.py

```diff
@@ -0,0 +1,29 @@
+def Fact(n):
+    '''
+        Fact function has one argument, which should be a natural number.
+        The function counts given number's factorial.
+    '''
+    if n == 0 or n == 1:
+        return 1
+    if n > 1:
+        return Fact(n - 1) * n
+
+def Pascal(n):
+    '''
+        Pascal function has one argument, which should be a natural number (n).
+        The function prints out the first n rows of Pascal's triangle.
+    '''
+    for i in range(0, n+1):
+        for j in range(0, i+1):
+            k = Fact(i)/(Fact(j) * Fact(i-j))
+            print(k, end = ' ')
+        print("\n")
+
+while True:
+    try:
+        n = int(input("Enter the number of row: "))
+        print("First " + str(n) + " rows of Pascal's triangle: ")
+        Pascal(n)
+    except ValueError:
+        print("Entered wrong value. Please enter a number.")
+        print()
```

Comment:
1. why double?
![image](https://user-images.githubusercontent.com/80816144/119723578-a8ad0480-be7e-11eb-8e34-f70bfb68a5b1.png)

2. Fix for big number (e.g 1000000)
![image](https://user-images.githubusercontent.com/80816144/119723704-cb3f1d80-be7e-11eb-85df-4d3fd215c3b9.png)

Reply:
again fix styles based on pep8